vulkaninfo/vulkaninfo.cpp

```diff
@@ -935,21 +934,28 @@ const char *help_message_body =
     "[-o <filename>, --output <filename>]\n"
     "                    Print output to a new file whose name is specified by filename.\n"
     "                    File will be written to the current working directory.\n"
-    "[--text]            Produce a text version of " APP_SHORT_NAME " output to stdout. This is\n"
+    "[--text]            Produce a text version of " APP_SHORT_NAME
+    " output to stdout. This is\n"
```

Comment:
are these (and below) `clang-format` changes?  They seem unrelated to the intended changes.

Reply:
yes, this one sneaked in under the radar.

Reply:
Looked through for accidental formatting changes and reverted them - I suspect my local clang-format is "different" and don't want to think about formatting changes today.